### ISSUE TITLE
Just doesn't work (Invalid argument passed: /usr/bin/node)

### ISSUE THREAD

**Issue by sidreshot** [jiahaog/nativefier#1083]
**Homework**

- [x] I looked at `nativefier --help` and https://github.com/jiahaog/nativefier/blob/master/docs/api.md
- [x] I searched existing issues, open & closed. Yes, my bug is new.
- [x] I'm using the latest version available at https://github.com/jiahaog/nativefier/releases

**Bug description**

Just doesn't work.
Any appeal is responsible
`Invalid argument passed: /usr/bin/node .
Nativefier supports short options (like "-n") and long options (like "--name"), all lowercase. Run "nativefier --help" for help.
Aborting
`

**Steps to reproduce**

```
nativefier --verbose "medium.com"
`Invalid argument passed: /usr/bin/node .
Nativefier supports short options (like "-n") and long options (like "--name"), all lowercase. Run "nativefier --help" for help.
Aborting
`
`nativefier --help
Invalid argument passed: /usr/bin/node .
Nativefier supports short options (like "-n") and long options (like "--name"), all lowercase. Run "nativefier --help" for help.
Aborting
`
```

**Actual behavior**

The program was expected to work

**Debug info**
`npm -v
6.14.4
`
`nodejs -v
v10.19.0
`
`lsb_release -a
No LSB modules are available.
Distributor ID:	Linuxmint
Description:	Linux Mint 20
Release:	20
Codename:	ulyana
`

**Comment by ronjouch** [comment]
Thanks! Will push an emergency fix later today. In the meantime, please use version 11.0.2, installable with `npm install nativefier@11.0.2`

**Comment by ronjouch** [comment]
@sidreshot issue fixed in 42.0.1, available on npm. Thanks again for the heads up. I'll add an integration test later to catch this kind of early breakage.

**Comment by sidreshot** [comment]
> add

Problem solved. Thanks
